### PR TITLE
sphinx documentation: first commit

### PR DESCRIPTION
**Pull request by katyakats**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,9 +26,11 @@ integration:
 env:
 	pipenv install -r requirements.txt
 
+
```
Comment (gtopper) [inline Makefile]: Accidental newline.

File: Makefile [hunk 2]
```diff
@@ -37,3 +39,9 @@ dist: dev-env
 .PHONY: set-version
 set-version:
 	python set-version.py
+
+.PHONY: html-docs
+html-docs: ## Build html docs
```
Comment (gtopper) [inline Makefile]: I know I'm nitpicking but what's with the double #?

Comment (gtopper) [reply Makefile]: Also I think just docs would be a better name than html-docs.

Comment (katyakats) [reply Makefile]: the double # are from mlrun :)))

File: docs/index.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
+
+
```
Comment (gtopper) [inline docs/index.rst]: Accidental newlines?

File: Makefile [hunk 4]
```diff
@@ -29,6 +29,7 @@ env:
 .PHONY: dev-env
 dev-env: env
 	pipenv install -r dev-requirements.txt
+	pipenv install -r docs/requirements.txt
```
Comment (gtopper) [inline Makefile:32]:
```
pipenv install -r dev-requirements.txt -r docs/requirements.txt
```

**Review by gtopper — APPROVED** [review]

**Pull request merged**